bitcoin/src/internal_macros.rs

```diff
@@ -252,6 +252,7 @@ macro_rules! define_extension_trait {
             fn $fn:ident$(<$($gen:ident: $gent:path),*>)?($($params:tt)*) $( -> $ret:ty )? $body:block
         )*
     }) => {
+        #[doc(notable_trait)]
```

Comment:
```suggestion
        #[cfg_attr(docsrs, doc(notable_trait))]
```

Ignore the other suggested change it only compiles on nightly.

Reply:
Ah thanks man!